src/plugins/unified_histogram/public/types.ts

```diff
@@ -72,9 +68,9 @@ export interface UnifiedHistogramChartLoadEvent {
    */
   adapters: UnifiedHistogramAdapters;
   /**
-   * Observable of the lens embeddable output
+   * Observable for the data change subscription
    */
-  embeddableOutput$?: Observable<LensEmbeddableOutput>;
+  dataLoading$?: PublishingSubject<boolean | undefined>;
```

Comment:
I've kept the previous behaviour where only on full load the `EmbeddableOutput$` was passed.
I think it makes sense to always passed it but kept the previous behaviour from before fro now.

Reply:
I was curious about this too because I didn't know how `dataLoading$` was used. Looks like it's passed to the Lens edit flyout here: https://github.com/elastic/kibana/blob/06c715d854b8a3a5529267e7ef4801ab335b78ef/src/plugins/unified_histogram/public/chart/chart_config_panel.tsx#L103-L110